src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFileActionLink.kt

```diff
@@ -18,7 +20,20 @@ class ContextFileActionLink(
 
   init {
     text = contextItemFile.getLinkActionText(project.basePath)
-    toolTipText = contextItemFile.uri.path
+    font =
+        when {
+          contextItemFile.isIgnored == true || contextItemFile.isTooLarge == true ->
+              Font(
+                  super.getFont().attributes +
+                      (TextAttribute.STRIKETHROUGH to TextAttribute.STRIKETHROUGH_ON))
+          else -> super.getFont()
+        }
+    toolTipText =
+        when {
+          contextItemFile.isIgnored == true -> "File ignored by an admin setting"
+          contextItemFile.isTooLarge == true -> "Excluded due to context window limit"
```

Comment:
so it was a valid scenario that we displayed a context file used while actually it has been ignored due to the size, right? then this was missing - thanks for fixing btw! 🙇‍♂️

Reply:
No worries, I was just following along from the VSCode implementation.